src/test/ui/closure-tuple-diagnostics.rs

```diff
@@ -0,0 +1,15 @@
+#![feature(capture_disjoint_fields)]
+//~^ WARNING: the feature `capture_disjoint_fields` is incomplete
+//~| `#[warn(incomplete_features)]` on by default
+//~| see issue #53488 <https://github.com/rust-lang/rust/issues/53488>
+struct S(String);
+
+fn expect_fn<F: Fn()>(_f: F) {}
+
+fn main() {
+    let s = S(format!("s"));
+    let c = || { //~ ERROR expected a closure
+        let s = s.0;
```

Comment:
Then use s.1

Reply:
I have tried it, but it doesn't seem like a valid thing to do